docs/generated/metrics/metrics.html

```diff
@@ -996,8 +996,10 @@
 <tr><td>APPLICATION</td><td>distsender.errors.notleaseholder</td><td>Number of NotLeaseHolderErrors encountered from replica-addressed RPCs</td><td>Errors</td><td>COUNTER</td><td>COUNT</td><td>AVG</td><td>NON_NEGATIVE_DERIVATIVE</td></tr>
 <tr><td>APPLICATION</td><td>distsender.rangefeed.catchup_ranges</td><td>Number of ranges in catchup mode<br/><br/>This counts the number of ranges with an active rangefeed that are performing catchup scan.<br/></td><td>Ranges</td><td>GAUGE</td><td>COUNT</td><td>AVG</td><td>NONE</td></tr>
 <tr><td>APPLICATION</td><td>distsender.rangefeed.error_catchup_ranges</td><td>Number of ranges in catchup mode which experienced an error</td><td>Ranges</td><td>COUNTER</td><td>COUNT</td><td>AVG</td><td>NON_NEGATIVE_DERIVATIVE</td></tr>
+<tr><td>APPLICATION</td><td>distsender.rangefeed.fatal_errors</td><td>Number of rangefeeds that were stopped due to fatal errors</td><td>Ranges</td><td>COUNTER</td><td>COUNT</td><td>AVG</td><td>NON_NEGATIVE_DERIVATIVE</td></tr>
 <tr><td>APPLICATION</td><td>distsender.rangefeed.local_ranges</td><td>Number of ranges connected to local node.</td><td>Ranges</td><td>GAUGE</td><td>COUNT</td><td>AVG</td><td>NONE</td></tr>
 <tr><td>APPLICATION</td><td>distsender.rangefeed.restart_ranges</td><td>Number of ranges that were restarted due to transient errors</td><td>Ranges</td><td>COUNTER</td><td>COUNT</td><td>AVG</td><td>NON_NEGATIVE_DERIVATIVE</td></tr>
+<tr><td>APPLICATION</td><td>distsender.rangefeed.retry.eof</td><td>Number of ranges that encountered retryable eof error</td><td>Ranges</td><td>COUNTER</td><td>COUNT</td><td>AVG</td><td>NON_NEGATIVE_DERIVATIVE</td></tr>
```

Comment:
These metric names are a bit confusing to me. They appear to count number of rangefeeds, but the naming suggests that we're counting errors.

Reply:
It's not just the names that are confusing. The metrics are a little confusing.  In some cases, one error can generate restarts of all active rangefeeds that are part of this mux range feed. Really what I think this is aiming to count is "rangefeeds restarted by reason"